packages/serverpod_serialization/lib/src/websocket_messages.dart

```diff
@@ -4,6 +4,29 @@ import 'package:serverpod_serialization/serverpod_serialization.dart';
 
 /// Base class for messages sent over a WebSocket connection.
 sealed class WebSocketMessage {
+  /// The version of the protocol.
+  static const int version = 1;
+
+  /// The keyword used for the version in the websocket message.
+  static const String messageVersionKeyword = 'version';
```

Comment:
Should this really be on a per-message level? The protocol version is not likely to change during a session. Perhaps it would be better with a first message setting the expectations for the protocol version?

Using this approach, it would be possible to add this feature at a later stage too, by assuming the version is the first version if there is no first versioning message.

Reply:
I removed the version from the messages and will only keep the improved formatting of the message and some test improvements.